templates/Gebruiker/index.html.twig

```diff
@@ -61,8 +61,8 @@
                         <td>
                             {{ gebruiker.type|transform_type_in_title }}
                         </td>
-                        <td>{{ gebruiker.teamGka.naam }}</td>
-                        <td>{{ gebruiker.schuldhulpbureaus|join(', ') }}</td>
+                        <td>{% if gebruiker.teamGka|default %}{{ gebruiker.teamGka.naam }}{% endif %}</td>
+                        <td>{% if gebruiker.schuldhulpbureaus|default %}{{ gebruiker.schuldhulpbureaus|join(', ') }}{% endif %}</td>
```

Comment:
Beetje minor, maar ik vind het leesbaarder als je dit indent. 
Weet je ook zeker dat je de cell leeg wilt laten als `gebruiker.teamGka` geen waarde heeft?

Reply:
Ik heb hier gekozen om het bestaande gedrag van de applicatie zo veel mogelijk in stand te houden. Gezien de originele code zal de applicatie hier altijd al een leeg veld getoond hebben als er geen waarde was, maar omdat dit niet expliciet was gaf dit in developmentomgevingen een foutmelding (ik ben niet zeker of dit voor de update van symfony ook het geval was, maar na de update werd dit in ieder geval afgedwongen in developmentomgevingen).